packages/synthetic-chain/package.json

```diff
@@ -25,6 +25,7 @@
     "@endo/zip": "^1.0.1",
     "better-sqlite3": "^9.4.0",
     "chalk": "^5.3.0",
+    "cosmjs-types": "^0.9.0",
```

Comment:
Why prefer `cosmjs-types` to `@agoric/cosmic-proto` or `@cosmology/telescope`?

Reply:
fewer build layers